core/templates/pages/story-editor-page/editor-tab/story-editor.directive.ts

```diff
@@ -344,6 +346,13 @@ angular.module('oppia').directive('storyEditor', [
             }
           };
 
+          $scope.addFocusWithoutScroll = function(label) {
+            FocusManagerService.setFocus(label);
+            setTimeout(function() {
+              window.scrollTo(0, 0);
+            }, 5);
```

Comment:
Hm if we prevent scrolling, would it be a bit confusing to the user if the focused element is outside the view port?

Reply:
I have removed it from here, as the content above is added by the user while creating the story before, But at some place we are preventing it to scroll so that user doesn't miss reading the neccesary info. especially in small screens.